CMakeLists.txt

```diff
@@ -222,6 +222,46 @@ else (MINGW OR APPLE)
       SET(Mscore_SHARE_NAME    "share/")
 endif (MINGW OR APPLE)
 
+# Download MuseScore SoundFont
```

Comment:
@lasconic Please put **the entire block** into something that can be disabled, for example with `-DDOWNLOADABLE_SOUNDFONT=OFF` on the cmake command line.

Something like this (internet access during the build) is absolutely untenable for almost all packaging systems and distributions.

In Debian, I’ve already started packaging the soundfont separately (from the `.sf2` source), so not having the soundfont in the MuseScore repo won’t affect us, but adding this _will_.

Reply:
OK will do.

Reply:
Done. Rebased. It can be called on the CMake command line `-DDOWNLOAD_SOUNDFONT=OFF` but also via the main Makefile `make  DOWNLOAD_SOUNDFONT=OFF`